app/views/transition_landing_page/_buckets.html.erb

```diff
@@ -15,7 +14,6 @@
       } %>
     </div>
   </div>
-
   <% campaign_links = I18n.t("transition_landing_page.campaign_links") %>
   <%
     links = campaign_links[:links].map do |link|
```

Comment:
Ah @injms this is as the link items are being built higher up using `map` so that it has the relevant elements.  Potentially this could be merged 🤔

Reply:
Makes sense, done 👍🏼